app/styles/recap-situation.css

```diff
@@ -0,0 +1,66 @@
+.recap-situation {
+    border-top: none;
+    background-color: white;
+}
+
+.recap-situation h3 {
+    margin-top: 0;
+}
+
+.recap-situation .section-header {
+    margin-top: 21px;
+}
+
+.accordion-ressources .panel {
+    border: none;
+    font-size: 12px;
+}
+
+.accordion-ressources .panel-heading {
+    padding: 5px 8px;
+}
+
+.accordion-ressources .panel-title {
+    font-size: 14px;
+}
+
+.accordion-ressources .panel-body {
+    padding: 10px;
+}
+
+.accordion-ressources hr {
+    margin-top: 5px;
+    margin-bottom: 5px;
+}
+
+.accordion-ressources .recap-indemnites {
+    border-left: 7px solid #AC6C82;
```

Comment:
You can factorise the `7px solid` by not using the shorthand property.

First, use one selector that is valid for all the `.recap-` panels. If there is none, introduce one (possibly replacing `class="recap-indemnites"` by `class="ressource-recap indemnites"`).
In CSS, declare for this class: `border-style: solid; border-width: 7px;` (and put it in `em` along the way).
Then, you only have to specify `border-color` for each colour variant 😃